projects/plugins/crm/modules/woo-sync/admin/settings/main.page.php

```diff
@@ -305,7 +305,12 @@ function jpcrm_settings_page_html_woosync_main() {
 						<td class="wfieldname" colspan="2">
 							<label><?php esc_html_e( 'Order status map', 'zero-bs-crm' ); ?>:</label><br />
 							<?php esc_html_e( 'Here you can choose how you want to map WooCommerce order statuses to CRM statuses (if the above setting is enabled)', 'zero-bs-crm' ); ?>
-							<br/>
+							<div style="margin-top:10px;">
+								<a style="margin-top:10px;" href="<?php echo esc_url( $zbs->urls['woomanagingorders'] ); ?>" target="_blank">
+									<?php esc_html_e( 'Learn more about the WooCommerce order statuses', 'zero-bs-crm' ); ?>
+									<img class="jpcrm-external-link-icon" style="margin-bottom:2px" src="<?php echo esc_url( ZEROBSCRM_URL ); ?>i/external-link.svg" />
+								</a>
```

Comment:
I noticed that the spacing between the Learn more link and the external link icon is slightly larger than the spacing that is showing in the other example where this icon is used, within the learn more pop-up. Then I noticed that the text for the link is being output with a space on either side of it. 
When I concat the lines that spacing disappears. For example:

```suggestion
								<a style="margin-top:10px;" href="<?php echo esc_url( $zbs->urls['woomanagingorders'] ); ?>" target="_blank"><?php esc_html_e( 'Learn more about the WooCommerce order statuses', 'zero-bs-crm' ); ?><img class="jpcrm-external-link-icon" style="margin-bottom:2px" src="<?php echo esc_url( ZEROBSCRM_URL ); ?>i/external-link.svg" /></a>
```

Still approving as it's a minor thing.